helper/memory_calculator_test.go

```diff
@@ -142,7 +149,23 @@ func testMemoryCalculator(t *testing.T, context spec.G, it spec.S) {
 				})
 			})
 
-			it("limits total memory to 1G if unset", func() {
+			it("limits total memory to all available memory if no memory limit set", func() {
+				const s = `
+					MemTotal:       16400152 kB
+					MemFree:        10477724 kB
+					MemAvailable:   11514136 kB
+					Buffers:          112396 kB
+				`
+
+				Expect(ioutil.WriteFile(memoryLimitPath, strconv.AppendInt([]byte{}, helper.UnsetTotalMemory, 10), 0755)).To(Succeed())
+				Expect(ioutil.WriteFile(memoryInfoPath, strconv.AppendQuote([]byte{}, s), 0755)).To(Succeed())
```

Comment:
```suggestion
				Expect(ioutil.WriteFile(memoryInfoPath, []byte(s), 0755)).To(Succeed())
```
I don't think we want to double quote this.